### PR TITLE
Add new question: Gradient Clipping (L2)

### LINKED ISSUE #479: New Problems: Gradient ops based problems
I think it would be cool to also have the core gradient operation based problems like:
* Gradient Clipping
* Gradient Accumulation
* Gradient Checkpointing

Can work on these too @moe18

### PR DESCRIPTION
**Pull request by komaksym**
Resolves #479

### CODE REVIEW

**Comment by komaksym** [comment]
Hey @moe18, check these out whenever you get a minute, thanks! 
This and #546
